sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -876,6 +876,24 @@ class AnalysisErrorSuite extends AnalysisTest {
       "[`a`, `b`, `c`, `d`, `e`]"
       :: Nil)
 
+  errorTest(
```

Comment:
Could you invoke `errorClassTest()`. This will allow to make the test independent from error message text, so, tech editors could edit `error-classes.json` and don't depend on Spark's tests.

Reply:
done